### DISCUSSION TITLE
Does Rectangle Pro support URL actions?

### DISCUSSION THREAD

**Discussion by montemartin** [rxhanson/RectanglePro-Community#218] [General]
I'm doing a trial of Rectangle Pro and I'm in love with Stash mode. 

However I'm also trying to use rectangle URL actions along with [Bunch](https://bunchapp.co/) to automate window layouts. When I active a URL action it opens Rectangle even when Pro is already running. Is it still necessary to keep the free version running to use continue using URL actions?

**Comment by rxhanson** [comment]
Rectangle Pro has the base url `rectangle-pro`, rather than `rectangle`, so just switch over your URL calls and you won't need Rectangle. I'll be updating the Rectangle Pro site with some more documentation soon. Until then, I added a few bits on the root Github page, and there's a little more detail in this discussion: https://github.com/rxhanson/RectanglePro-Community/discussions/98

**Comment by montemartin** [comment]
Wow, thanks. I didn't even realize custom was possible. I'm coming from years using Mosaic and this feels so much more efficient.
